packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -498,6 +498,12 @@ export class LoggingError extends Error {
         props?: ITelemetryProperties,
     ) {
         super(message);
+        if (props) {
+            this.addProperties(props);
+        }
+    }
+
+    public addProperties(props: ITelemetryProperties) {
```

Comment:
Yay!  I have this same change pending [here](https://github.com/microsoft/FluidFramework/pull/5308/files#diff-a8e8af708cd24804ad20446f3a666813d33871f415efed5b116b64a381f93298R592-R598), but I called it `addTelemetryProperties` for symmetry with `getTelemetryProperties`, what do you think?